src/pages/admin/reports/components/program-expenses-section/components/common/add-program-expense-record-modal/AddProgramExpenseRecordModal.module.scss

```
@@ -0,0 +1,315 @@
+    align-items: center;
+    gap: 10px;
+}
+
+.exchange-rate-chip-label {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.1px;
+
+    color: c.$grey-900;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)